File: src/main/java/io/github/icusystem/icu_connect/api_icu/ModeSet.java
package io.github.icusystem.icu_connect.api_icu;
import java.util.ArrayList;
import java.util.List;

public class ModeSet {

    public List<ModeCamera> Cameras = new ArrayList<>();

    public ModeSet() {
        ModeCamera cam1 = new ModeCamera();
        if (Cameras != null) {
            Cameras.add(cam1);
        }
    }


}

File: src/main/java/com/itl/icu_connect/api_icu/ModeSet.kt
package com.itl.icu_connect.api_icu

class ModeCamera{
    var Id_Index:Int = 0
    var Face_rec_en:Boolean = false
    var Spoof_level:Int = 0
    var Pose_filter:Boolean = false
    var Camera_distance:Int = 0
    var Rotation:Int = 0
    var View_mode:String? = null
}

class ModeSet {

    var Cameras:MutableList<ModeCamera>? = ArrayList()

    init {
        val cam1 = ModeCamera()
        Cameras?.add(cam1)
    }

}

